tests/bud.bats

```diff
@@ -1641,3 +1641,34 @@ load helpers
   touch "$DIR"/Dockerfile
   run_buildah 0 bud "$DIR"/Dockerfile
 }
+
+@test "bud-no-change" {
+  parent=alpine
+  target=no-change-image
+  buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/no-change
+  run_buildah --log-level=error inspect --format '{{printf "%q" .FromImageDigest}}' ${parent}
+  parentid="$output"
+  run_buildah --log-level=error inspect --format '{{printf "%q" .FromImageDigest}}' ${target}
+  [[ "$output" == "$parentid" ]]
+
+  buildah rmi ${target}
+}
+
+@test "bud-no-change-label" {
+  parent=alpine
+  target=no-change-image
+  buildah bud --label "test=label" --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/no-change
+  run_buildah --log-level=error inspect --format '{{printf "%q" .Docker.Config.Labels}}' ${target}
+  expect_output 'map["test":"label"]'
```

Comment:
I suspect this will work now.  If by chance it doesn't try:
`  [[ "$output" == 'map["test":"label"]' ]]`

Similar suggestion below